### PR TITLE
fix(form): properly support displaying `helperText` for custom compon…

### PR DESCRIPTION
**Pull request by Kiarokh**
…ents

fix https://github.com/Lundalogik/crm-feature/issues/2582

## Review:
- [x] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [x] Commits have the correct *type* for the changes made
- [x] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

**Comment by Kiarokh** [comment]
this is actually done by @Befkadu1 ;)

**Comment by adrianschmidt** [comment]
I'm guessing this is what's missing when I'm not getting the helper text for the icon-picker in Lime Admin, while reviewing that PR? @Kiarokh

File: src/components/form/fields/schema-field.ts [hunk 1]
```diff
@@ -150,7 +150,9 @@ export class SchemaField extends React.Component<FieldProps> {
         const { errorSchema, schema } = this.props;
 
         if (!this.isInvalid()) {
-            return schema.description;
+            const helperText = schema.lime?.component?.props?.helperText;
+
+            return schema.description || helperText;
```
Comment (adrianschmidt) [inline src/components/form/fields/schema-field.ts:155]: If `helperText` has been explicitly set, shouldn't we prefer showing that over `schema.description` if both exist?

```suggestion
            return helperText || schema.description;
```

Comment (Kiarokh) [reply src/components/form/fields/schema-field.ts:155]: makes sense

**Review by adrianschmidt — APPROVED** [review]

**Pull request merged**